### DISCUSSION TITLE
Hi there

### DISCUSSION THREAD

**Discussion by RPRX** [XTLS/Xray-core#51] [General]
Welcome

**Comment by xianren78** [comment]
first

**Comment by RPRX** [comment]
#50 直接被 GitHub 跳过了，可能是因为我先点了 `Set up discussions` 却没有发布，等下去反馈

刚刚在 Xray-examples 试了 https://github.com/XTLS/Xray-examples/discussions/5 ，没有被跳过

**Comment by ixxmu** [comment]
讨论区正式开启！

**Comment by simpleandstupid** [comment]
ᕙ(＠°▽°＠)ᕗ

**Comment by evan1936** [comment]
真的很快

**Comment by badO1a5A90** [comment]
讨论区就是....水区?

**Reply by RPRX** [reply]
> 
> 
> 讨论区就是....水区?

不不不，还是 Project X 群每天几千条比较厉害

**Comment by tpfeng** [comment]
滴，打卡
